MetaMorpheus/EngineLayer/FdrAnalysis/FdrAnalysisEngine.cs

```diff
@@ -223,35 +188,31 @@ public static void Compute_PEPValue_Based_QValue(List<SpectralMatch> psms)
                 psms[psmsArrayIndicies[i]].FdrInfo.PEP_QValue = Math.Round(qValue, 6);
             }
         }
-
-        public void CountPsm()
+        /// <summary>
+        /// This method gets the PSM count for each psm with q-value < 0.01 and adds that information to the individual psm.
```

Comment:
I don't understand this summary comment. What is the PSM count for an individual psm? Is it always 1 or 0?

Reply:
comment updated w/ new explanation

![image](https://github.com/smith-chem-wisc/MetaMorpheus/assets/16841846/562d3706-ae5c-43a4-921e-64d9e7645881)